src/carbonio-files-ui-common/views/components/List.preview.test.tsx

```diff
@@ -71,8 +71,8 @@ describe('Preview action', () => {
 		expect(screen.queryByRole('button', { name: /download file/i })).not.toBeInTheDocument();
 	});
 
-	test('check if the context preview action is supported by mime type', async () => {
-		const files = populateNodes(4, 'File') as File[];
+	test('check if the context preview action is supported by mime type, and check that only docs has edit action in preview', async () => {
```

Comment:
If there is an and, it probably makes sense to split the test in 2 different ones. 
Don't follow the wrong example of the too big tests we created at the time :wink: 

To make the test even smaller, we could split the test in two, one to check those mime types where the action should be visible, one for those which should not have it. For each one of the two tests, we could then use an .each to reduce the lines of the test to the bare minimum